master/pkg/tasks/copy.go

```diff
@@ -22,7 +24,8 @@ const (
 
 func harnessArchive(harnessPath string, aug *model.AgentUserGroup) container.RunArchive {
 	var harnessFiles archive.Archive
-	wheelPaths, err := filepath.Glob(filepath.Join(harnessPath, "*.whl"))
+	validWhlNames := fmt.Sprintf("*%s*.whl", version.Version)
+	wheelPaths, err := filepath.Glob(filepath.Join(harnessPath, validWhlNames))
 	if err != nil {
 		panic(errors.Wrapf(err, "error finding Python wheel files in path: %s", harnessPath))
```

Comment:
non-blocking: add in the the version we are looking for into the error message

Reply:
yeah sure, I thought about it but it seemed redundant since the det version should be obvious, but couldn't hurt.